src/EditorFeatures/Core/Implementation/IntelliSense/ViewTextSpan.cs

```diff
@@ -118,7 +118,7 @@ private static IEnumerable<Span> MapUpToSnapshotRecursive(SnapshotSpan start, IP
         private static Span MapDownToSnapshot(Span span, IProjectionSnapshot start, ITextSnapshot target)
         {
             var sourceSpans = new Queue<SnapshotSpan>(start.MapToSourceSnapshots(span));
-            while (true)
+            while (sourceSpans.Count > 0)
```

Comment:
I'm now confused by this change -- doesn't this just mean we replace one exception (the .Dequeue on empty queue) with the Contract.Fail now?